tests/containers/kubectl.pm

```diff
@@ -0,0 +1,69 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test the kubectl utility
+# - List pods
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use utils;
+use version_utils;
+use publiccloud::utils;
+use registration qw(cleanup_registration register_product add_suseconnect_product get_addon_fullname remove_suseconnect_product);
+use containers::k8s;
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    # kubectl is in the container module (on SLES)
+    add_suseconnect_product(get_addon_fullname('contm')) if (is_sle);
+    zypper_call("-n in kubernetes-client");
```

Comment:
Should we use the function install_kubectl that from k8s library?

Reply:
Good point, I will move it there and also add the container module addition to the library function